packages/manager/apps/hub/webpack.config.js

```diff
@@ -17,8 +18,16 @@ module.exports = (env = {}) => {
     REGION ? Object.assign(env, { region: REGION }) : env,
   );
 
+  // Extra config files
+  const extrasRegion = glob.sync(`./.extras-${REGION}/**/*.js`);
```

Comment:
What are the extra folders for ?

Reply:
@FredericEspiau It has been put in place in order to hook into it at the build phase.
For instance we can plug Raven, AT-Internet smarttag, Chatbot etc.. :)

Reply:
ok they are generic folders then, got it !

Reply:
Update was made on `develop` https://github.com/ovh/manager/pull/2276